cli/client/check.go

```diff
@@ -50,19 +50,8 @@ func (client *RestClient) UpdateCheck(check *types.CheckConfig) (err error) {
 }
 
 // DeleteCheck deletes check from configured Sensu instance
-func (client *RestClient) DeleteCheck(check *types.CheckConfig) error {
-	path := checksPath(client.config.Namespace(), check.Name)
-	res, err := client.R().Delete(path)
-
-	if err != nil {
-		return err
-	}
-
-	if res.StatusCode() >= 400 {
-		return UnmarshalError(res)
```

Comment:
I don't think `UnmarshalError()` is in any potential code path anymore after these changes. Could that change the CLI's output in the face of some errors?

Reply:
I'm not sure what you mean.. the generic `Delete` function could still reach the `UnmarshalError`: https://github.com/sensu/sensu-go/blob/6780d12f05bca9c296e95ca99688a17b9b1e7eb3/cli/client/generic.go#L18

Reply:
Indeed! Sorry that's my bad.